packages/orchestration/src/examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Examples
+
+This directory contains sample contracts showcasing the Orchestration API. Each example demonstrates specific functionalities:
+
+- **stakeIca.contract.js**: Interchain account creation for remote staking
```

Comment:
This uses a low-level API. It should either be updated or moved to a separate list or something.

It does seem to have an end-to-end test, so updating it would be nice.
[stake-ica.test.ts](https://github.com/Agoric/agoric-sdk/blob/master/multichain-testing/test/stake-ica.test.ts)

Reply:
I think it would be a good exercise for me to update it :)